props/saber_fett263_buttons.h

```diff
@@ -0,0 +1,307 @@
+// Fett263 Buttons
+// Includes "Multi-Blast" Mode to enable Swing Blast control and SA22C volume menu
+//
+// 2 Button Controls (PWR and AUX):
```

Comment:
It's hard to know how similar/different this is from the other props...
Do you have a list of what the actual changes are?